fetch.bs

```diff
@@ -4593,6 +4593,31 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    using <var>request</var>.
   </ol>
 
+ <li>
+  <p>If <var>response</var>'s <a for=response>status</a> is 421, then:
+
+  <ol>
+   <li>
+    <p>If the ongoing fetch is <a for=fetch>terminated</a>, then:
+
+    <ol>
+     <li><p>Let <var>aborted</var> be the termination's aborted flag.
+
+     <li><p>If <var>aborted</var> is set, then return an <a>aborted network error</a>.
+
+     <li><p>Return a <a>network error</a>.
+    </ol>
+
+   <li>
+    <p>If <var>actualResponse</var>'s <a for=response>status</a> is 421,
+     <var>request</var>'s <a for=request>body</a> is non-null, and
+     <var>request</var>'s <a for=request>body</a>'s <a for=body>source</a> is
+      null, then return a <a>network error</a>.
+
+   <li><p>Set <var>response</var> to the result of performing an <a>HTTP-network-or-cache fetch</a>
+   using <var>request</var>.
```

Comment:
The p
```suggestion
   using <var>request</var> on a new <a lt=connection>connection</a>.
```

The point of 421 is that the new request will be on a separate connection.

Reply:
Oh I see, that seems important to call out indeed and might require more changes than this.

In particular this would need to invoke https://fetch.spec.whatwg.org/#concept-connection-obtain but add some argument to bypass the cache lookup I suppose.

Reply:
How about do HTTP-network fetch instead of HTTP-network-or-cache fetch to skip cache ?
Also since HTTP-network fetch includes obtaining a connection, "on a new connection" declaration is really needed here ?

Reply:
I'm not talking about the HTTP cache, I'm talking about the connection pool "cache". This needs to ensure you get a new connection if I understand it correctly, not reuse an existing one. (It's probably fine to invoke HTTP-network fetch directly, but you'd still need to signal that it needs a fresh connection, perhaps with a new argument in a few places.)

Reply:
I see.
Added a description about the new connection.

Reply:
I think we should address this in the way I suggested. Whereby obtain a connection has a new argument that forces the lookup to be bypassed so a new connection is returned. Trying to clarify this in the caller's prose doesn't make it clear that this is a feature an "obtain a connection" implementation needs to provide.

Reply:
O.K. Updated "obtain a connection" with new argument.